index.js

```diff
@@ -18,9 +23,33 @@ module.exports = {
     }
   },
 
+  treeForVendor(defaultTree) {
+    var trees = [];
+
+		if (defaultTree) {
+			trees.push(defaultTree);
+    }
+    
+    let trackJsPath = path.join(this.project.root, 'node_modules', 'trackjs');
```

Comment:
I was under the impression we were supposed to be using `require.sync` or `require.resolve` to get the path of the file?

Reply:
In funnel we need path to the folder and not the file.

Reply:
I imagine this is good enough for now, but I think this may complicate things for people using yarn workspaces (that said, I don't use them, so I'm only guessing).

Let's try this and see if anyone runs into issues.